dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -182,18 +228,42 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 
 					if (collection.collectionType === 'fixed/thrasher') {
 						return (
-							<Section
-								fullWidth={true}
-								padSides={false}
-								padBottom={true}
-								showTopBorder={false}
-								showSideBorders={true}
-								ophanComponentLink={ophanComponentLink}
-								ophanComponentName={ophanName}
-								containerName={collection.collectionType}
-							>
-								<Snap snapData={trails[0].snapData} />
-							</Section>
+							<>
+								<Section
+									fullWidth={true}
+									padSides={false}
+									padBottom={true}
+									showTopBorder={false}
+									showSideBorders={true}
+									ophanComponentLink={ophanComponentLink}
+									ophanComponentName={ophanName}
+									containerName={collection.collectionType}
+								>
+									<Snap snapData={trails[0].snapData} />
+								</Section>
+								{shouldInsertMerchHigh(
+									index,
+									front.isNetworkFront,
+									front.pressedPage.collections.length,
+									front.pressedPage.frontProperties
+										.isPaidContent,
+								) && (
+									<AdSlot
+										data-print-layout="hide"
+										position="merchandising-high"
+										display={format.display}
+									/>
+								)}
```

Comment:
Was the absence of potentially inserting the merchandising high slot here a bug we're fixing alongside adding mobile ads? If so it could be worth calling this our separately in the PR description!